docs/finetune.md

```diff
@@ -46,21 +46,30 @@ Training:
     GPU: 1
   accelerate_mode: GPU_DDP
 ```
+For HPU, set `device` to HPU, set HPU number in `resources_per_worker` and set `accelerate_mode` to HPU_DDP or HPU_DEEPSPEED.
+```
+Training:
+  device: HPU
+  resources_per_worker:
+    CPU: 1
+    HPU: 1
+  accelerate_mode: HPU_DDP
```

Comment:
I am wondering if we can just use DDP，FSDP, and Deepspeed for this parameter without adding CPU_, GPU_ and HPU_. Because we can already know the device from `device` parameter, right?

Reply:
yes. Refactoring